src/portals/blockchain-frontend/apps/blockchain-frontend/app/(authenticated)/trainers/page.tsx

```diff
@@ -1,11 +1,13 @@
 "use client";
 import styles from './cart-page-trainers.module.css';
-import {Form} from 'antd'
+import {Form,Table} from 'antd'
 import Input from 'antd/es/input';
 import Button from 'antd/es/button';
-import {PlusOutlined} from '@ant-design/icons/lib/icons'
+import {AccountBookFilled, PlusOutlined} from '@ant-design/icons/lib/icons'
 import axios from 'axios';
 import React,{useState} from 'react';
+import { randomFill } from 'crypto';
```

Comment:
what is the package  'crypto' ?

Reply:
It is not use in the code, I removed.. Done